internal/notifier/discord.go

```diff
@@ -64,6 +64,11 @@ func NewDiscord(hookURL string, username string, channel string) (*Discord, erro
 
 // Post Discord message
 func (s *Discord) Post(event recorder.Event) error {
+	// Skip any update events
+	if status, ok := event.Metadata["commit_status"]; ok && status == "update" {
```

Comment:
Can we please make this into a function `IsCommitStatus` and reuse it in all providers.